src/client/features/enrichment-ui.js

```diff
@@ -0,0 +1,40 @@
+const React = require('react');
```

Comment:
you should move all of your ui components to a singular folder under /features.  Probably in ```features/enrichment``` folder because I think that is the overarching concept.